hack/test-integration.sh

```diff
@@ -28,7 +28,9 @@ tags="${OS_TEST_TAGS:-integration no-docker}"
 
 export GOMAXPROCS="$(grep "processor" -c /proc/cpuinfo 2>/dev/null || sysctl -n hw.logicalcpu 2>/dev/null || 1)"
 TMPDIR=${TMPDIR:-/tmp}
-export BASETMPDIR=${BASETMPDIR:-"${TMPDIR}/openshift-integration"}
+export BASETMPDIR=${BASETMPDIR:-"${TMPDIR}/openshift-integration"} 
+
+#"
```

Comment:
My bad guys, the sublime colorization gets jacked up by line 31, that was my workaround to be able to edit the rest of the file without going blind.  I had it committed over here but not pushed.  You aren't crazy.